Source/bmalloc/bmalloc.xcodeproj/project.pbxproj

```diff
@@ -643,6 +643,7 @@
 		6599C5CD1EC3F15900A2F7BB /* AvailableMemory.h in Headers */ = {isa = PBXBuildFile; fileRef = 6599C5CB1EC3F15900A2F7BB /* AvailableMemory.h */; settings = {ATTRIBUTES = (Private, ); }; };
 		7939885B2076EEB60074A2E7 /* BulkDecommit.h in Headers */ = {isa = PBXBuildFile; fileRef = 7939885A2076EEB50074A2E7 /* BulkDecommit.h */; settings = {ATTRIBUTES = (Private, ); }; };
 		795AB3C7206E0D340074FE76 /* PhysicalPageMap.h in Headers */ = {isa = PBXBuildFile; fileRef = 795AB3C6206E0D250074FE76 /* PhysicalPageMap.h */; settings = {ATTRIBUTES = (Private, ); }; };
+		7B9D3EA129953B0E001C869C /* BAllowProhibitedAPIs.h in Headers */ = {isa = PBXBuildFile; fileRef = 7B9D3EA029953B0E001C869C /* BAllowProhibitedAPIs.h */; settings = {ATTRIBUTES = (Private, ); }; };
```

Comment:
I think this can be a project header, since it's only included by a source file.